src/main/java/pages/ResizablePage.java

```diff
@@ -22,38 +27,31 @@ public ResizablePage(WebDriver driver) {
     public void assertThatSizeOfElementWasChanged() {
         WebElement elementAfterResize = getDriver().findElement(By.cssSelector("#resizable"));
         Dimension elementSize = elementAfterResize.getSize();
-        System.out.println(elementSize.getWidth());
-        System.out.println(elementSize.getHeight());
-
+        System.out.println("End width: " + elementSize.getWidth());
+        System.out.println("End height: " + elementSize.getHeight());
         LOG.info("Assert that Wight is the same");
-        Assert.assertEquals(183, elementSize.getWidth());
+        Assert.assertEquals("Width is wrong after resize", (startWidthElement + OFFSET - anchorWidth - 1), elementSize.getWidth());
```

Comment:
assertTrue, assertEquals - first parameter is always error message that we will see in case of failure. This is very important to have readable test result.

I changed hardcoded value of 183 to dynamic value that is calculated during the test. 
So, the main idea is to take the size on the beginning. Increase this size to OFFSET value. But! We need to decrease for the value of anchor element that we're using for resizing. Because this anchor element is inside of our element. Decreasing to -1 - is only derivation of calculation that happens after conversion from float to int, etc.